src/resources/views/auth/list.blade.php

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>クイズ一覧</title>
+    </head>
+    <body>
+      @extends('layouts.app')
+
+      @section('content')
+        @foreach ($links as $link)
+        <div style="text-align: center">
+          <a href="quiz/{{$link->id}}">{{$link->name}}</a>
```

Comment:
routeヘルパ使ったほうが良いと思います
```suggestion
          <a href="route()">{{$link->name}}</a>
```